### ISSUE TITLE
Retrying setup: Subscripted generics cannot be used with class and instance checks

### ISSUE THREAD

**Issue by cloudbr34k84** [xannor/ha_reolink_rest#26]
I just updated to the latest version and im getting this when im adding IP Devices
Retrying setup: Subscripted generics cannot be used with class and instance checks

I have the E1 Outdoor. They were working fine a version back fyi

**Comment by xannor** [comment]
Can you load the full log and provide the stack trace so I can see in my code where this is triggering.

**Comment by cloudbr34k84** [comment]
Hey Xannor, thanks fopr replying mate

 I wasnt sure what you needed when i first logghed the issue

Here is the trace

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 205, in _async_refresh
    self.data = await self._async_update_data()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 164, in _async_update_data
    return await self.update_method()
  File "/config/custom_components/reolink_rest/__init__.py", line 82, in _update_data
    return await entity_data.async_update()
  File "/config/custom_components/reolink_rest/entity.py", line 475, in async_update
    result = await self._execute_commands(commands, command_channel=command_channel)
  File "/config/custom_components/reolink_rest/entity.py", line 335, in _execute_commands
    or self._process_ptz_responses(
  File "/config/custom_components/reolink_rest/entity.py", line 690, in _process_ptz_responses
    self.ptz[channel].update_presets(response.presets)
  File "/config/custom_components/reolink_rest/entity.py", line 217, in update_presets
    if value is not None and not isinstance(value, Mapping[int, ptz.Preset]):
  File "/usr/local/lib/python3.10/typing.py", line 994, in __instancecheck__
    return self.__subclasscheck__(type(obj))
  File "/usr/local/lib/python3.10/typing.py", line 997, in __subclasscheck__
    raise TypeError("Subscripted generics cannot be used with"
TypeError: Subscripted generics cannot be used with class and instance checks
```

**Comment by kroven9** [comment]
I've also encountered an issue with an E1 camera. I was able to add all my other POE cameras but my wireless E1 Zoom is having issues. Basically when I configure it it asks for my admin login but then skips adding it to area saying its finished. The camera then reports an error.

**Comment by xannor** [comment]
I published a new release v0.6.7 let me know if this works for you.

**Comment by kroven9** [comment]
> I published a new release v0.6.7 let me know if this works for you.

E1 Zoom is now working properly for me.
